app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -197,15 +196,14 @@ class HomeFragment : Fragment() {
 
         sessionControlView = SessionControlView(homeFragmentStore, view.homeLayout, sessionControlInteractor)
 
-        view.homeLayout.applyConstraintSet {
-            sessionControlView.view {
-                connect(
-                    TOP to BOTTOM of view.wordmark_spacer,
-                    START to START of PARENT_ID,
-                    END to END of PARENT_ID,
-                    BOTTOM to TOP of view.bottom_bar
-                )
-            }
+        val constraintSet = ConstraintSet()
+        constraintSet.apply {
+            clone(view.homeLayout)
+            connect(sessionControlView.view.id, TOP, view.wordmark_spacer.id, BOTTOM)
+            connect(sessionControlView.view.id, START, ConstraintSet.PARENT_ID, START)
```

Comment:
To be consistent, `PARENT_ID` should probably be used without the prefix as well
```suggestion
            connect(sessionControlView.view.id, START, PARENT_ID, START)
```